spec/schema.json

```diff
@@ -436,29 +436,60 @@
         }
       },
       "OverseasEntityDueDiligence": {
+        "required": [
+          "name",
+          "email"
+        ],
         "type": "object",
         "properties": {
           "identity_date": {
             "type": "string",
             "format": "date"
           },
           "name": {
-            "type": "string"
+            "type": "string",
+            "allOf" : [
+              {
+                "$ref": "#/components/schemas/FieldPattern"
+              },
+              {
+                "maxLength": 256
+              }
+            ]
           },
           "identity_address": {
             "$ref": "#/components/schemas/Address"
           },
           "email": {
-            "type": "string"
```

Comment:
Probably better if we create a component schema for email and use `$ref`

Reply:
Lets do it when we have consistency between emails, already discussed with Francis to check why we have got some fields with length 256 and other 250 when the table can accept 256.